### PR TITLE
New test refactor

### PR DESCRIPTION
**Pull request by sara-02**
With changes in PySpark path and test recommendation validation.

### CODE REVIEW

File: tests/unit_tests/test_until_error_codes.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from unittest import TestCase
+
+
+class TestUtilInputValidation(TestCase):
+
+    def test_input_invalid_error(self):
+        try:
+            raise AnalyticsException(error_codes.ERR_INPUT_INVALID)
+        except AnalyticsException as e:
+            print e
```
Comment (fridex) [inline tests/unit_tests/test_until_error_codes.py]: What is tested here? (Also it would be great to make this python3 compliant)

Comment (sara-02) [reply tests/unit_tests/test_until_error_codes.py]: @fridex The `Util/ErrorCode` class is tested here, have updated this to assert the error codes.

**Comment by vpavlin** [comment]
[test]

**Comment by sara-02** [comment]
@fridex @vpavlin  The build is failing because of this change https://github.com/fabric8-analytics/fabric8-analytics-stack-analysis/pull/63

**Comment by sara-02** [comment]
[test]

**Comment by vpavlin** [comment]
[test]

**Comment by sara-02** [comment]
Task done as a part of https://github.com/openshiftio/openshift.io/issues/974

**Pull request merged**